packages/browser-wallet/src/inject/index.ts

```diff
@@ -1,8 +1,10 @@
+/* eslint-disable no-console */
 import { IWalletApi, walletApi } from '@concordium/browser-wallet-api';
-import { logger } from '@concordium/browser-wallet-message-hub/src/message-handlers/logger';
 
 // Inject WalletAPI to dApp
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 (window as any).concordium = walletApi as IWalletApi;
 
-logger.log('::InjectedScripted executed');
+// TODO figure out best way to let DOM know that the API is ready for use. This assumes consumers declare a global function "concordiumReady"
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+(window as any).concordiumReady();
```

Comment:
Is when the wallet API is ready? (I thought it would just be when window.concordium wasn't undefined? (but I guess this is to have a callback?)

Reply:
Yes, otherwise the consumer would have to periodically check - I think this is more elegant, but open to other suggestions. You can check `example.html` how it's used.

Reply:
The only issue I have is that if the wallet is ready before the consumer is ready, would this not give an error?
(and then when they define it afterwards, it wouldn't get triggered?)